packages/ui/src/form-elements/image-upload/index.tsx

```diff
@@ -95,10 +95,26 @@ const ImageUploadField: FC<ImageUploadProps> = ({
         ? allowedTypes
         : "";
 
+
+    useEffect(() => {
+        setTimeout(() => {
+            const inputItem = document.querySelectorAll('.filepond--browser');
+            const label = document.querySelectorAll('.filepond--drop-label > label');
+            inputItem.forEach((item: any) => {
+                item.setAttribute('tabindex', '-1');
+            });
+            label.forEach((item: any) => {
+                item.setAttribute('aria-hidden', 'false');
+            });
+
+         }, 500);
```

Comment:
Je weet nu nog niet 100% zeker dat filepond geladen is, op een tragere computer / trager internet is dat niet gegarandeerd na 500ms in de `useEffect`, toch? Ik denk dat een check met bijv. MutationObserver wat degelijker is; a la https://stackoverflow.com/a/61511955